### PR TITLE
Print summaries to stdout instead of stderr 

### PR DESCRIPTION
**Pull request by stweil**
Signed-off-by: Stefan Weil <sw@weilnetz.de>

### CODE REVIEW

**Comment by zuphilip** [comment]
Thank you very much! It looks that this will solve #132 in these two files.

File: ocropus-econf [hunk 1]
```diff
@@ -84,11 +84,11 @@ for fname,e,t,m,c in outputs:
 if perfile is not None: perfile.close()
 if allconf is not None: allconf.close()
 
-sys.stderr.write("errors    %8d\n"%errs)
-sys.stderr.write("missing   %8d\n"%missing)
-sys.stderr.write("total     %8d\n"%total)
-sys.stderr.write("err       %8.3f %%\n"%(errs*100.0/total))
-sys.stderr.write("errnomiss %8.3f %%\n"%((errs-missing)*100.0/total))
+print "errors    %8d"%errs
```
Comment (QuLogic) [inline ocropus-econf]: Use `print()` and be Python 3 ready. There's only one argument, so there's no effect on Python 2.

Comment (stweil) [inline ocropus-econf]: Good hint. I'll add a commit which fixes that tomorrow.

**Comment by amitdo** [comment]
Stefan, please add `from __future__ import print_function`

File: ocropus-econf [hunk 2]
```diff
@@ -84,14 +84,14 @@ for fname,e,t,m,c in outputs:
 if perfile is not None: perfile.close()
 if allconf is not None: allconf.close()
 
-print "errors    %8d"%errs
-print "missing   %8d"%missing
-print "total     %8d"%total
-print "err       %8.3f %%"%(errs*100.0/total)
-print "errnomiss %8.3f %%"%((errs-missing)*100.0/total)
+print("errors    %8d"%errs)
```
Comment (amitdo) [inline ocropus-econf]: Stefan, please add from __future__ import print_function on top

Comment (stweil) [inline ocropus-econf]: Is it also needed if print only gets one argument? At least for Python 2.7.9 it works without that statement.

Comment (zuphilip) [inline ocropus-econf]: Maybe not needed here, but we will add that statement anyway during PR #155 see https://github.com/tmbdev/ocropy/pull/155/files#diff-672c98336ceb05ab25dc90ed9934175f (I hope we can continue on these other PR the next weeks)

Comment (zuphilip) [inline ocropus-econf]: Thus, IMO we can add this statement also afterwards when we deal with the other occurrences of printing statements.

Comment (amitdo) [inline ocropus-econf]: I thought it will fail in 2.6/2.7... but you reminded me that in this case it's fine.

**Comment by zuphilip** [comment]
@stweil FYI I just tried out the solve merge conflicts here on GitHub and it worked well.

**Comment by zuphilip** [comment]
Thank you very much @stweil !

**Pull request merged**